resources/htmlTemplate/TocHTMLTemplate.html

```diff
@@ -178,7 +177,7 @@ <h1>SPDX License List</h1>
       <hr />
       <h2>Deprecated Licenses</h2>
       <p>
-      Release 2.0 of the SPDX Specification introduces <strong><a href="https://spdx.org/specifications">License Expression Syntax</a></strong> that supports the ability to identify common variations of SPDX-identified licenses without the need to define each potential variation as a distinct license on the SPDX License List. This new syntax supports the ability to use a simple “+” operator after a license short identifier to indicate “or later version” (e.g. GPL-2.0+). It also supports the ability to declare an SPDX-identified <a href="./exceptions-index.html"><strong>license exception</strong></a> using the “WITH” operator (e.g. GPL-2.0+ WITH Autoconf-exception-2.0). SPDX has defined a list of license exceptions to use after the “WITH” operator. As a result, a number of licenses formerly included in the SPDX License List have been deprecated as licenses, and correct usage employs the License Expression Syntax as of v2.0. The URL to each deprecated license still exists and those license pages have been updated to note the deprecation. Other licenses may have been deprecated for reasons noted.
+      Release 2.0 of the SPDX Specification introduced <strong><a href="https://spdx.org/specifications">License Expressions</a></strong> that supports the ability to identify common variations of SPDX-identified licenses without the need to define each potential variation as a distinct license on the SPDX License List. This new syntax supports the ability to declare an SPDX-identified <a href="./exceptions-index.html"><strong>license exception</strong></a> using the “WITH” operator (e.g. GPL-2.0+ WITH Autoconf-exception-2.0), as well as the ability to use a simple “+” operator after a license short identifier to indicate “or later version”. SPDX has defined a list of license exceptions to use after the “WITH” operator. As a result, a number of licenses formerly included on the SPDX License List have been deprecated, and correct usage employs the License Expression syntax as of v2.0. The URL to each deprecated license still exists and those license pages have been updated to note the deprecation. Other licenses may have been deprecated for the reasons noted.
```

Comment:
With `GPL-2.0+` deprecated (spdx/license-list-XML#543), you should probably update the `GPL-2.0+ WITH Autoconf-exception-2.0` example here.  I'd also consider wrapping it in `<code>…</code>` for styling that sets it off from the main text.